larky/src/test/resources/vgs_tests/nts/test_default_nts.star

```diff
@@ -0,0 +1,120 @@
+load("@vendor//asserts", "asserts")
+load("@stdlib//unittest", "unittest")
+load("@vgs//nts", "nts")
+
+
+def _test_render():
+    input = {
+        "merchantAccount": "YOUR_MERCHANT_ACCOUNT",
+        "reference": "YOUR_PAYMENT_REFERENCE",
+        "amount": {
+            "currency": "USD",
+            "value": 1000,
+        },
+        "paymentMethod": {
+            "type": "networkToken",
+            "holderName": "CARDHOLDER_NAME",
+            "number": "785840aLpH4nUmV9985",
+            "expiryMonth": "TO_BE_REPLACED",
+            "expiryYear": "TO_BE_REPLACED",
+        },
+        "returnUrl": "https://your-company.com/",
+        "shopperReference": "YOUR_SHOPPER_REFERENCE",
+        "recurringProcessingModel": "CardOnFile",
+        "shopperInteraction": "Ecommerce",
+        # Deep JSONPath is not supported by the JSONPath lib, so that we need to
+        # create an empty object here manually.
+        # ref: https://github.com/json-path/JsonPath/issues/83
+        "mpiData": {
+            "cavv": "TO_BE_REPLACED",
+            "eci": "TO_BE_REPLACED",
+        }
+    }
+
+    output = nts.render(
```

Comment:
This is great, but I have some thoughts.
1. Can we return an object that the Larky script can use to input the values itself, rather than relying on the templatized rendering method? For example:
```
network_token_data = nts.render(input['paymentMethod']['number'])
input['mpiData']['cavv'] = network_token_data.cryptogram
input['mpiData']['eci'] = network_token_data.eci
input['paymentMethod']['expiryYear'] = network_token_data.expiry_year
input['paymentMethod']['expiryMonth'] = network_token_data.expiry_month
input['paymentMethod']['number'] = network_token_data.DPAN
```
With the current approach, I'm worried about any complex integrations. I'm not sure how this approach would work with form data, XML, graphql, etc... By providing the data right to the integration, I think the integration level code will be easier to implement in each circumstance.

2. If we do stick with the templatized rendering approach, we need to ensure that the end-user doesn't need to send the "TO_BE_REPLACED" fields. If the field doesn't exist, I would expect it to be auto-created. Again need to account for how this would work with a graphql API (which I recently had to work with, hence being top of mind :) )